### DISCUSSION TITLE
Search comments

### DISCUSSION THREAD

**Discussion by megacuber** [CubeLabsNZ/CubeTime#194] [General]
Hi! Can the Search feature on the Times screen search for text in the comments of solves, or just search for numbers within the times? Thanks!

**Comment by xbjfk** [comment]
I added this in 806fdafdeda0a277445d727a3361f465192d4fd3 :)

**Reply by megacuber** [reply]
I saw that in the update! Thanks for all the time and care you've put into such a great app!
